crates/oxc_codegen/src/sourcemap_builder.rs

```diff
@@ -62,12 +62,12 @@ impl SourcemapBuilder {
     }
 
     pub fn add_source_mapping_for_name(&mut self, output: &[u8], span: Span, name: &str) {
-        // SAFETY: search original string by span.
-        let original_name =
-            unsafe { self.original_source.get_unchecked(span.start as usize..span.end as usize) };
+        // Here using `get_unchecked` happened crash at windows. See https://github.com/rolldown/rolldown/issues/716
```

Comment:
Maybe we could add a `debug_assert!(span.end < self.original_source.len())` to deny all possiable invalid span in debug build. And and silent the error in release build because nothing we could do in that time.